doc/2018/2018-07-31.md

```diff
@@ -0,0 +1,111 @@
+# Hello Ubuntu Server
+
+The purpose of this communication is to provide a status update and
+highlights for any interesting subjects from the Ubuntu Server Team. If
+you would like to reach the server team, you can find us at
+the #ubuntu-server channel on Freenode. Alternatively, you can sign up
+and use the Ubuntu Server Team [mailing list](https://lists.ubuntu.com/mailman/listinfo/ubuntu-server).
+
+## Spotlight:
+The Ubuntu Server Installer (subiquity-based) has been released in Bionic
+Ubuntu Server ISOs. It now carries additional configuration support for:
+ - LVM
+ - RAID
+ - VLAN
+ - Bonds.
+
+There is a [call for testing](https://lists.ubuntu.com/archives/ubuntu-server/2018-July/007725.html) for these features in Ubuntu.  [Josh Powers](https://github.com/powersj) on Canonical's ubuntu server team has [a blog post explaining with details about the new installer features](https://powersj.github.io/post/ubuntu-bionic-subiquity-updated/).
+
```

Comment:
Can you change the first URL to:
https://powersj.io

And my blog URL to 
https://powersj.io/post/ubuntu-bionic-subiquity-updated/

Reply:
done. thanks